README.md

```diff
@@ -231,12 +238,12 @@ You can change the number of HugePages available to each MarkLogic container by
 
 MarkLogic Server Docker containers ship with a small set of scripts, making it easy to create clusters. See the [MarkLogic documentation](https://docs.marklogic.com/guide/concepts/clustering) for more about clusters. The following three examples show how to create MarkLogic Server clusters with Docker containers. The first two use Docker compose scripts to create one-node and three-node clusters. See the documentation for [Docker compose](https://docs.docker.com/compose/) for more details. The third example demonstrates a container setup on separate VMs.
 
-The credentials for the admin user are configured using Docker secrets, and are stored in `mldb_admin_username.txt` and `mldb_admin_password.txt` files.
+The credentials for the admin user are configured using Docker secrets, and are stored in `mldb_admin_username.txt`, `mldb_admin_password.txt` and `mldb_wallet_password.txt` files.
```

Comment:
add a comma after `mldb_admin_password.txt`  

 `mldb_admin_password.txt`, and `mldb_wallet_password.txt` files.

Reply:
Fixed.